crates/gen/src/types/type_name.rs

```diff
@@ -425,183 +431,21 @@ fn format_abi_ident(name: &str) -> proc_macro2::Ident {
     quote::format_ident!("abi_{}", name)
 }
 
-#[cfg(test)]
```

Comment:
Same thought here. These tests have helped me in the past.

Reply:
Fair enough. Restored all tests.